hw/ip/otbn/dv/model/otbn_model.h

```diff
@@ -12,7 +12,7 @@
 
 #include "otbn_memutil.h"
 
-class ISSWrapper;
+struct ISSWrapper;
```

Comment:
Wow! I'm surprised that neither my local Clang nor GCC complained about this :-)